### PR TITLE
Manual Update of PGO/IBC Data

### PR DESCRIPTION
**Pull request by brianrob**
Updating to latest data from 2020/08/06.

cc: @AndyAyersMS, @kunalspathak, @billwert, @davidwrighton, @mangod9

### CODE REVIEW

File: eng/Version.Details.xml [hunk 1]
```diff
@@ -66,25 +66,25 @@
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>f6192d1e284a08ac05041d05fa6e60dec74b24f5</Sha>
     </Dependency>
-    <Dependency Name="optimization.windows_nt-x64.IBC.CoreFx" Version="99.99.99-master-20190716.1">
+    <Dependency Name="optimization.windows_nt-x64.IBC.CoreFx" Version="99.99.99-master-20200806.6">
```
Comment (mangod9) [inline eng/Version.Details.xml:69]: so this implies these havent been updated for more than a year?

Comment (brianrob) [reply eng/Version.Details.xml:69]: These were last updated with data from February 28, 2020.  You can see that in the change to version.props.  Version.Details.xml is used to keep track of changes that come in via dependency flow.  I just didn't update Version.Details.xml last time around.

**Comment by mangod9** [comment]
`runtime-live-build` seems to be failing with this:
```
src\coreclr\src\.nuget\optdata\optdata.csproj(0,0): error NU1102: Unable to find package optimization.windows_nt-x64.PGO.CoreCLR with version (>= 99.99.99-master-20200806.6)
  - Found 414 version(s) in dotnet-eng [ Nearest version: 99.99.99-master-20200301.3 ]
```

File: eng/Version.Details.xml (hunk 1 above)
Comment (mangod9) [reply eng/Version.Details.xml:69]: ok, thanks for the info.

**Comment by brianrob** [comment]
Yup, I missed one of the packages when getting them uploaded.  It should be there now, and I just re-triggered the failing runs.

**Review by mangod9 — APPROVED** [review]
Thanks for updating these!

**Comment by BruceForstall** [comment]
@brianrob There's no automated (or semi-automated) system to update these?

**Comment by brianrob** [comment]
There was, but as the engineering infrastructure evolved, the optimization infrastructure hasn't been updated.  This is on the list of things that need to be done, but for now, we'll do manual updates as necessary.

**Comment by BruceForstall** [comment]
Can I assume (hope) that we'll get updated data for rc1 and rc2 such that we're shipping in 5.0 the most optimized code?

**Comment by brianrob** [comment]
Historically, we have continued generating new data until we fork for release, and so expect to have fresh data for that.  We can also consider generating new data for the RCs, but if they're already forked, there is likely a decent amount of work to be done there, and so we'd want to know that the code churn is enough to warrant the work.

**Comment by BruceForstall** [comment]
The last fork happened on Monday, to release/5.0 branch. This will be the source of RC1, RC2, and the final release.

**Comment by brianrob** [comment]
@BruceForstall, I've done a bit of cleanup in the data generation, and forked it for 5.0.  So, we do have the ability to replace the data if we need to.

**Comment by BruceForstall** [comment]
@brianrob Does the build process (or any other process) give us any indication of how "good" the data is, and when it might need to be regenerated? I recall that some systems like this would output a "% of matching basic blocks", for instance, and that number would degrade quickly with code changes.

**Comment by davidwrighton** [comment]
No, this is one of the details that might be interesting to add to any new work on our PGO solution for managed code.

**Comment by brianrob** [comment]
Right, it's possible to get this kind of data from native PGO on Windows.  It is not possible from IBC, and I don't believe it's possible with the Linux PGO toolset.

**Pull request merged**